src/resdk/resolwe.py

```diff
@@ -97,9 +97,11 @@ class Resolwe:
     group = None
     feature = None
     mapping = None
+    session = None
```

Comment:
Can you add an empty line above? Now, it appears as session is one of the resources...

Reply:
Sure. Actually I added session here just to make tests pass.
I can also remove it and add it in test methods.